src/games/snake/board.py

```diff
@@ -0,0 +1,48 @@
+from random import randrange
+import pygame
+from src.games.snake.fruit import Fruit
+
+
+class Board():
+    def __init__(self, width, height, fields_horizontally, fields_vertically):
+        self._fields_horizontally = fields_horizontally
+        self._fields_vertically = fields_vertically
+
+        self._field_width = int(width / fields_horizontally)
+        self._field_height = int(height / fields_vertically)
+        self._grid = [[pygame.Rect(x * self._field_width, y * self._field_height, self._field_width, self._field_height)
+                        for y in range(fields_vertically)]
+                            for x in range(fields_horizontally)]
```

Comment:
these lines are not intended as PEP8 suggests. It should be:
```suggestion
                       for y in range(fields_vertically)]
                      for x in range(fields_horizontally)]
```
I agree with PEP because it's more readable for me with that convention.